server/driver.go

```diff
@@ -70,6 +70,12 @@ type QueryCtx interface {
 	// GetStatement gets PreparedStatement by statement ID.
 	GetStatement(stmtID int) PreparedStatement
 
+	// StoreResultSet stores ResultSet for subsequent using.
+	StoreResultSet(stmtID int, rs ResultSet)
```

Comment:
The `ResultSet` is associated with a `PreparedStatement`.
I think it is better to add the method to `PreparedStatement` instead of QueryCtx.